src/features/datamodel/useBindingSchema.tsx

```diff
@@ -68,7 +70,15 @@ function getDataModelUrl({
   isAnonymous,
   isStateless,
   instanceId,
+  prefillFromQueryParams,
 }: DataModelDeps & DataModelProps) {
+  if (prefillFromQueryParams && isStateless && !isAnonymous && dataType) {
```

Comment:
I don't see why we should limit this not to work for anonymous stateless? :thinking: